automation/test.sh

```diff
@@ -9,6 +9,6 @@ trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
 
 make cluster-down
 make cluster-up
-make cluster-sync-handler test/local/e2e
-make cluster-sync
-make test/cluster/e2e
+#TODO: Not working at stdci
```

Comment:
please add WIP or /hold unless we resolve these

Reply:
It's the local testing, we can merge without it since we don't know if we are going to have it at distributed.

Reply:
Going to remove test/local/e2e is working